recipes/bioconductor-bambu/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
     - 'https://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
     - 'https://bioarchive.galaxyproject.org/{{ name }}_{{ version }}.tar.gz'
     - 'https://depot.galaxyproject.org/software/bioconductor-{{ name }}/bioconductor-{{ name }}_{{ version }}_src_all.tar.gz'
-  md5: ca673a3a01222cd203b4234c71098032
+  md5: 969fb4fa7e4e2959276c1c7de7b243a9
 build:
   number: 2
```

Comment:
This should be reset to 0 since you've bumped it to a newer version 
Lint error:
```
ERROR: recipes/bioconductor-bambu/meta.yaml:15: build_number_needs_reset: The recipe build number should be reset to 0
```